<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1554</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VkViz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  padding: 5px;
  border: 2px solid black;
  font: 16pt 'Monospace';
}

QFrame {
  padding: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="Splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="BufferLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>888</width>
             <height>739</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	max-height: 30px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="CmdBufferTab">
               <attribute name="title">
                <string>Submitted Commands</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QTreeWidget" name="CmdBufferTree">
                  <property name="font">
                   <font>
                    <family>DejaVu Sans Mono</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="rootIsDecorated">
                   <bool>true</bool>
                  </property>
                  <attribute name="headerVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Submitted Commands</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="ResourceTab">
               <attribute name="title">
                <string>Resource View</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QWidget" name="PickerWidget" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>743</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item alignment="Qt::AlignRight|Qt::AlignTop">
                    <widget class="QComboBox" name="ResourcePicker">
                     <property name="maximumSize">
                      <size>
                       <width>79</width>
                       <height>23</height>
                      </size>
                     </property>
                     <item>
                      <property name="text">
                       <string>Image 0</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Image 1</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Image 2</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Image 3</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QTreeWidget" name="SyncView">
                  <property name="headerHidden">
                   <bool>true</bool>
                  </property>
                  <column>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </column>
                  <item>
                   <property name="text">
                    <string>Columns</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Are</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Fun</string>
                    </property>
                   </item>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="ToggleCommands">
            <property name="font">
             <font>
              <family>Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Collapse Buffers</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="ResourceLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font-size: 14pt;
}</string>
            </property>
            <property name="text">
             <string>Capture Frame</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1554</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
